docs/migration-guides/prometheus-migration.md

```diff
@@ -0,0 +1,340 @@
+# Prometheus Time Series Database Migration
+
+If you already have a running Kubernetes cluster and you are interestd on installing the Bitnami Kubernetes Production Runtime (BKPR) on it you may want to migrate the persisted data of your services with zero downtime and a minimal data loss. In this case, we will talk about the Prometheus Time Series Data Base (TSDB), which contains the metrics that Prometheus scrapped from your services.
+
+This document describes the steps to migrate the TSDB from an existing Prometheus deployment to the one installed by BKPR.
+
+Furthermore, this process will not generate any downtime for your current Prometheus deployment, making it safe to be followed.
+
+## Prerequisites
+
+The following prerequisites should be met before starting with the migration process:
+
+* You have a healthy Kubernetes cluster running.
+* You have a Prometheus deployment that *matches the BKPR Prometheus version*, which can be obtained [here](https://github.com/bitnami/kube-prod-runtime#release-compatibility).
+
+This article will not cover the upgrade process of Prometheus itself. If you need to update your Prometheus deployment, check the [Prometheus docs](https://prometheus.io/docs/prometheus/latest/migration/)
+
+## Overview
+
+This section provides an overview of the required steps to migrate the Prometheus TSDB from an already running deployment to the BKPR Prometheus one.
+
+This process has been designed and tested to ensure the shortest possible downtime when migrating to the new StatefulSet by having two Prometheus instances running at the same time. These instances will be modified to include a sidecar container that will deploy an *rsync* client and server with access to the Prometheus data volume in order to move a Prometheus TSDB snapshot from one pod to another.
+
+Before starting the migration is recommended to get familiarized with the concepts of TSDB and Write-Ahead-Log (WAL) file. See the following links for more information about it:
+
+* [Prometheus TSDB](https://prometheus.io/docs/prometheus/latest/storage/)
+
+* [Write-Ahead-Log file](https://github.com/prometheus/tsdb/blob/master/docs/format/wal.md)
+
+As Prometheus doesn’t include yet a *flush* method to dump the WAL file that is being written into a snapshot with a block format that doesn’t overlap time ranges (see [this issue](https://github.com/prometheus/tsdb/issues/346) for more context) it is necessary to sync with the Prometheus TSDB compaction times, in order to ensure a minimal data loss (less than 5 minutes).
+
+These are the steps required to perform this migration:
+
+* Deploy an rsync server-side container (acting as the rsync server) in your current Prometheus deployment
```

Comment:
In some places, rsync is in italics and in some it is in normal text. Could you make this consistent? I think the italics version is better, because rsync is a command name.

Reply:
done